src/sentry/ratelimits/cardinality.py

```diff
@@ -132,3 +157,266 @@ def use_quotas(
             indicates how much quota should actually be consumed.
         """
         pass
+
+
+class RedisCardinalityLimiter(CardinalityLimiter):
+    """
+    The Redis cardinality limiter stores a key per unit hash, and adds the unit
+    hash to a set key prefixed with `RequestedQuota.prefix`. The memory usage
+    grows with the dimensionality of `prefix` and the `unit_hashes` that fit
```

Comment:
I think you mean cardinality here, not dimensionality. unit_hashes is a set. The number of elements of a set is its cardinality.

Reply:
sorry yes dimensionality was only supposed to refer to `prefix` (as an input value) which is also present outside of the set in the key names. I slightly altered the wording, perhaps it's clearer now.